dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/LoopUnrolling.java

```diff
@@ -289,7 +289,7 @@ private Map<Function, Map<CondJump, Integer>> loadLoopBoundsMapFromFile(Program
                         .put(loopJump, bound);
             }
         } catch (IOException e) {
-            e.printStackTrace();
+            // Nothing to be done, filePath is guaranteed to exists when the FileReader is called.
```

Comment:
Not sure if this is true. `IOExceptions` not only happen if the file is missing I think.

Reply:
You are right, I will handle this case as the other ones.